lib/code_corps/stripe_service/events/account_updated.ex

```diff
@@ -1,5 +1,5 @@
 defmodule CodeCorps.StripeService.Events.AccountUpdated do
-  def handle(%{"data" => %{"object" => %{"id" => id_from_stripe}}}) do
+  def handle(%{data: %{object: %{id: id_from_stripe}}}) do
```

Comment:
One of the changes involved passing in the fetched `Stripe.Event` into the handlers, instead of the raw json.